lib/data/network/interceptor/matrix_dio_cache_interceptor.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:dio/dio.dart';
+import 'package:dio_cache_interceptor/dio_cache_interceptor.dart';
+import 'package:matrix/matrix.dart';
+
+class MatrixDioCacheInterceptor extends DioCacheInterceptor {
+  MatrixDioCacheInterceptor({required super.options});
+
+  final List<String> _uriSupportsCache = [];
+
+  void addUriSupportsCache(List<String> uriSupportsCache) {
+    Logs().d(
+      'DioCacheCustomInterceptor::setPathSupportsCache() Add uriSupportsCache: $uriSupportsCache',
+    );
+    _uriSupportsCache.addAll(uriSupportsCache);
+    Logs().d(
+      'DioCacheCustomInterceptor::setPathSupportsCache() _pathSupportsCache: $_uriSupportsCache',
+    );
+  }
+
+  bool _cacheSupported(String currentPath) {
+    return _uriSupportsCache.any((path) => path.contains(currentPath));
```

Comment:
mean if _uriSupportsCache contains `vnexpress.net/tin-moi-nhat` then currentPath is `vnexpress.net`, it will return true ?